test/e2e/persistent_volumes.go

```diff
@@ -555,7 +559,7 @@ func makeWritePod(ns string, pvcName string) *api.Pod {
 					Name:    "write-pod",
 					Image:   "gcr.io/google_containers/busybox:1.24",
 					Command: []string{"/bin/sh"},
-					Args:    []string{"-c", "touch /mnt/SUCCESS && exit 0 || exit 1"},
+					Args:    []string{"-c", "touch /mnt/SUCCESS && (id -G | grep ' 777$') && exit 0 || exit 1"},
```

Comment:
yeah, assuming the image never changes and is in ascending order. I will use nicer regex to be safe